.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,68 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
+-->
+
+
+#### What type of PR is this?
+
+<!--
+Please label this pull request according to what type of issue you are
+addressing, especially if this is a release targeted pull request.
+
+Uncomment only one `/kind <>` line, hit enter to put that in a new line, and
+remove leading whitespace from that line:
 -->
+
+> /kind api-change
+> /kind bug
+> /kind cleanup
+> /kind deprecation
+> /kind design
+> /kind documentation
+> /kind failing-test
+> /kind feature
+> /kind flake
+> /kind other
```

Comment:
Podman doesn't actually have most of these labels. And TBH, the ones we do have (bug, cleanup, documentation) we don't seem to really use.

Reply:
Concur, this isn't really necessary IMO. Tagging issues is sufficient, we don't need to tag PRs.

Reply:
My understanding (could be wrong) is the bot will create and maintain the labels for us.  Whatever we write in the template is what will be used.  I think the purpose is to help at release-notes time, so you can filter out all the PRs having a docs or flakes etc.  In any case, I'm in agreement we should use the `/kind` list (and labels) that are most relevant/helpful.  Someone just needs to tell me what that set is, and I'll make it happen.

Note: Github treates PRs and Issues as the same kind of "thing".  The same set of labels can be applied to both, there's no such thing as PR or issue-only labels.